http/pagerenderer.go

```diff
@@ -66,13 +67,20 @@ func (w *RedirectRenderer) Write(ctx *Context, results *mirrors.Results) (status
 
 		path := strings.TrimPrefix(results.FileInfo.Path, "/")
 
-		// Generate the header alternative links
-		for i, m := range results.MirrorList[1:] {
-			var countryCode string
-			if len(m.CountryFields) > 0 {
-				countryCode = strings.ToLower(m.CountryFields[0])
+		mh := len(results.MirrorList)
+		if mh > GetConfig().MaxLinkHeaders+1 {
```

Comment:
Since GetConfig() holds a lock only for the current call, the value can therefore change during a reload. Thus you should get the value of MaxLinkHeaders only once.